presto-main/src/test/java/com/facebook/presto/sql/planner/sanity/TestVerifyOnlyOneOutputNode.java

```diff
@@ -41,7 +41,7 @@ public void testValidateSuccessful()
                                 new ValuesNode(
                                         idAllocator.getNextId(), ImmutableList.of(), ImmutableList.of(), ImmutableList.of()),
                                 Assignments.of()
-                        ), ImmutableList.of(), ImmutableList.of());
+                        ), ImmutableList.of(), ImmutableList.of(), ImmutableList.of());
```

Comment:
It's sad we had multiple params per line before. Let's fix it by having one param per line.

Reply:
There are way too many of these things. In general I agree we should change them, but I'd rather not taking on that part in this change.